app/components/ProductCard.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+import React from 'react';
+
+interface Product {
+  id: number;
+  title: string;
+  price: number;
+  image: string;
+}
+
+interface ProductCardProps {
+    product: Product;
+    currency: string; // e.g., 'USD', 'EUR', 'JPY'
+  }
```

Comment:
Do move the interfaces to a .d.ts file and rename interfaces. Also, do remove the comment.